hivemind/averaging/averager.py

```diff
@@ -539,7 +492,7 @@ async def _declare_for_download_periodically(self):
             await asyncio.sleep(self._matchmaking.averaging_expiration)
 
     async def rpc_download_state(
-        self, request: averaging_pb2.DownloadRequest, context: grpc.ServicerContext
+        self, request: averaging_pb2.DownloadRequest, _: P2PContext
```

Comment:
Same here with `_context`

Reply:
Still, seems slightly odd that in some cases this argument has a proper name and in some it doesn't

Reply:
Agreed.